src/tests/ftest/mpiio/hdf5.yaml

```
@@ -10,6 +10,8 @@ server_config:
         bdev_list: ["0000:81:00.0","0000:da:00.0"]
         scm_class: dcpm
         scm_list: ["/dev/pmem0"]
+      env_vars:
```

Spacing is off. Should be exactly 4 spaces
```suggestion
    env_vars:
```